pkgs/development/libraries/libpulsar/default.nix

```diff
@@ -0,0 +1,98 @@
+{ boost17x
+, clang
+, clang-tools
+, cmake
+, curl
+, enablePython ? false
+, fetchurl
+, gcc
+, gtest ? null
+, jsoncpp
+, lib
+, llvmPackages
+, log4cxx ? null
+, openssl
+, pkg-config
+, protobuf
+, python3 ? null
+, snappy ? null
+, stdenv
+, zlib ? null
+, zstd ? null
+}:
```

Comment:
From nixpkgs [conventions](https://nixos.org/manual/nixpkgs/stable/#chap-conventions)

_Arguments should be listed in the order they are used, with the exception of lib, which always goes first._

Reply:
I think the arguments are more closely ordered to how they are used now. Thank you!